content/en/docs/ops/diagnostic-tools/proxy-cmd/index.md

```diff
@@ -154,25 +154,39 @@ to send a request from the `productpage` pod to the `reviews` pod at `reviews:90
+    10.96.0.1     443   ALL                                              Cluster: outbound|443||kubernetes.default.svc.cluster.local
+    10.100.93.102 853   App: HTTP                                        Route: istiod.istio-system.svc.cluster.local:853
+    10.100.93.102 853   ALL                                              Cluster: outbound|853||istiod.istio-system.svc.cluster.local
+    0.0.0.0       9080  App: HTTP                                        Route: 9080
+    0.0.0.0       9080  ALL                                              PassthroughCluster
+    0.0.0.0       9090  App: HTTP                                        Route: 9090
+    0.0.0.0       9090  ALL                                              PassthroughCluster
+    10.96.0.10    9153  App: HTTP                                        Route: kube-dns.kube-system.svc.cluster.local:9153
+    10.96.0.10    9153  ALL                                              Cluster: outbound|9153||kube-dns.kube-system.svc.cluster.local
+    0.0.0.0       15001 ALL                                              PassthroughCluster
+    0.0.0.0       15006 Addr: 10.244.0.22/32:15021                       inbound|15021|mgmt-15021|mgmtCluster
+    0.0.0.0       15006 Addr: 10.244.0.22/32:9080                        Inline Route: /*
+    0.0.0.0       15006 Trans: tls; App: HTTP TLS; Addr: 0.0.0.0/0       Inline Route: /*
+    0.0.0.0       15006 App: HTTP; Addr: 0.0.0.0/0                       Inline Route: /*
+    0.0.0.0       15006 App: Istio HTTP Plain; Addr: 10.244.0.22/32:9080 Inline Route: /*
+    0.0.0.0       15006 Addr: 0.0.0.0/0                                  InboundPassthroughClusterIpv4
+    0.0.0.0       15006 Trans: tls; App: TCP TLS; Addr: 0.0.0.0/0        InboundPassthroughClusterIpv4
+    0.0.0.0       15010 App: HTTP                                        Route: 15010
+    0.0.0.0       15010 ALL                                              PassthroughCluster
+    10.100.93.102 15012 ALL                                              Cluster: outbound|15012||istiod.istio-system.svc.cluster.local
+    0.0.0.0       15014 App: HTTP                                        Route: 15014
+    0.0.0.0       15014 ALL                                              PassthroughCluster
+    0.0.0.0       15021 ALL                                              Inline Route: /healthz/ready*
+    10.111.121.13 15021 App: HTTP                                        Route: istio-ingressgateway.istio-system.svc.cluster.local:15021
+    10.111.121.13 15021 ALL                                              Cluster: outbound|15021||istio-ingressgateway.istio-system.svc.cluster.local
+    0.0.0.0       15090 ALL                                              Inline Route: /stats/prometheus*
+    10.111.121.13 15443 ALL                                              Cluster: outbound|15443||istio-ingressgateway.istio-system.svc.cluster.local
     {{< /text >}}
```

Comment:
I don't think it's sufficient to just change this output. Seems to me the surrounding text will need to be updated, since it isn't a correct anymore.

Reply:
I think the `MATCH` column is self-explanatory with more details as compare to previous version. It indicates `HTTP Plain`, `HTTP TLS`, `TCP TLS` and also displays complete address and ports. 

The surrounding text with custom markup will mess up with the indentation because of the new `DESTINATION` column which occupies most of the screen.

Reply:
Sorry, I meant generally, the surrounding text outside of the updated `{{< text >}}` sections. For example, just below it talks about `useOriginalDst`, which seems to be deprecated. It should probably try to be a little more clear about how to interpret the MATCH and DESTINATION rule fields, although in fairness, it wasn't very good at that before either.

Reply:
Gotcha, I will update the text providing info related to what `proxy-config listeners` output.

Reply:
Haven't found much info that can fit here. Only described what is `PassThroughCluster` and `Route`. @frankbu PTAL.